eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -709,17 +727,19 @@ object Transactions {
   val PlaceHolderSig = ByteVector64(ByteVector.fill(64)(0xaa))
   assert(der(PlaceHolderSig).size == 72)
 
-  private def sign(tx: Transaction, redeemScript: ByteVector, amount: Satoshi, key: PrivateKey, sighashType: Int): ByteVector64 = {
-    val sigDER = Transaction.signInput(tx, inputIndex = 0, redeemScript, sighashType, amount, SIGVERSION_WITNESS_V0, key)
+  private def sign(tx: Transaction, redeemScript: ByteVector, amount: Satoshi, key: PrivateKey, sighash: Int): ByteVector64 = {
```

Comment:
Why the name change? Isn't that the *type* of the `sighash`?

Reply:
I wanted to at least harmonize, it's true that `bitcoin-lib` calls it `sighashType`.
Bitcoin documentation calls it `sighash` or `sighash_flags`, but internally bitcoin core calls it `sighash_type`, so you're right I should update that back to `sighashType` everywhere!